flusher.go

```diff
@@ -67,7 +67,12 @@ func (s *Server) Flush(interval time.Duration, metricLimit int) {
 		totalLength += totalSets
 	}
 
+	// Iterate over all the metrics we gathered from the workers and combine them
+	// in to one final slice. We'll invoke `Flush` on each sampler which will
+	// spit out it's value!
 	combineStart := time.Now()
+	// allocating this long array to count up the sizes is cheaper than appending
+	// the metrics together one at a time
```

Comment:
bleh, github thinks this comment is outdated because the commit afterwards changed some of the trailing diff context